src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier ball = new ColorSupplier();
+
+    String getRandomBall() {
+        Random random = new Random();
+        return (random.nextInt(101) + "   " + ball.getRandomColor());
```

Comment:
You should return a Ball from this method:

```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(MAXIMUM_NUMBER) + 1);
```